src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -41,6 +41,29 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         return false;
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code word}.
+     *   Ignores case and finds partial match.
+     *   <br>examples:<pre>
+     *       containsWordIgnoreCase("ABc def", "abc") == true
+     *       containsWordIgnoreCase("ABc def", "DEF") == true
+     *       containsWordIgnoreCase("ABc def", "AB") == true (partial match)
+     *       containsWordIgnoreCase("ABc def", "ABCD") == false
+     *       </pre>
+     * @param sentence cannot be null
+     * @param word cannot be null, cannot be empty, must be a single word
+     */
+    public static boolean partiallyContainsWordIgnoreCase(String sentence, String word) {
```

Comment:
Change `word` to `toFind` or `toSearch`
Change `sentence` to more appropriate name (maybe, `fieldValue`)